ext/pdo_pgsql/tests/session_state_reset.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
+Persistent connections: session state reset when performing disconnect-equivalent processing (general case)
+--EXTENSIONS--
+pdo_pgsql
+--SKIPIF--
+<?php
+require __DIR__ . '/config.inc';
+require __DIR__ . '/../../../ext/pdo/tests/pdo_test.inc';
+PDOTest::skip();
+?>
+--FILE--
+<?php
+putenv('PDOTEST_ATTR='.serialize([PDO::ATTR_PERSISTENT => true]));
+
+require __DIR__ . '/../../../ext/pdo/tests/pdo_test.inc';
+
+$pdo1 = PDOTest::test_factory(__DIR__ . '/common.phpt');
+
+$pid1 = (int)$pdo1
+    ->query('select pg_backend_pid()::int;')
+    ->fetchColumn(0);
+
+$defaultValue = (int)$pdo1
+    ->query('show log_min_duration_statement;')
+    ->fetchColumn(0);
+
+$setValue = $defaultValue + 1;
+
+$pdo1->exec("set log_min_duration_statement = {$setValue};");
+
+$pdo1 = null;
+
+$pdo2 = PDOTest::test_factory(__DIR__ . '/common.phpt');
+
+$pid2 = (int)$pdo2
+    ->query('select pg_backend_pid()::int;')
+    ->fetchColumn(0);
+
+$expectedValue = (int)$pdo2
+    ->query('show log_min_duration_statement;')
+    ->fetchColumn(0);
+
+echo "pid1 is equals to pid2: " . (($pid1 === $pid2) ? 'yes' : 'no') . "\n";
```

Comment:
it is supposed to be yes, I would prefer then an assertion or `if (...) die(...);`

Reply:
I've switched to using `assert()`. I noticed other tests doing the same. Thanks for pointing that out!